actionpack/lib/action_dispatch/testing/integration.rb

```diff
@@ -283,6 +283,83 @@ def https?
         @https
       end
 
+      # Performs the actual request.
+      def process(method, path, params: nil, headers: nil, env: nil, xhr: false, as: nil)
```

Comment:
We should try fitting the documentation that's currently nested under `get` here.

As well as change the `Check #get for more details` to mention process instead.